public/js/atlas.js

```diff
@@ -523,19 +523,15 @@ function dateChangedString(site) {
 }
 
 function versionForSite(site) {
-    if (site.openmrs_version && site.openmrs_version !== "" && site.openmrs_version !== "unknown") {
-        var version = site.openmrs_version;
-        return version.match(/\d+(\.\d+)+/g).toString();
-    }
-    return null;
+    var version = site.openmrs_version;
+    version = version? version.match(/\d+(\.\d+)/g) : null;
```

Comment:
I believe your missing a `+` in your regex. This function returns the full version (not just major & minor version numbers). For example, given "1.2.3", this function should return "1.2.3". Look carefully at the regex in [this line](https://github.com/openmrs/openmrs-contrib-atlas/blob/master/public/js/atlas.js#L552).

```node
return version.match(/\d+(\.\d+)+/g).toString();
```

Note the `+` following the parenthetical, which means the `(\.\d+)` can repeat multiple times.

Reply:
@bmamlin Oops. Updated the PR! :)